blurring_as_a_service/inference_pipeline/submit_inference_pipeline.py

```diff
@@ -36,17 +29,26 @@ def main():
     aml_interface = AMLInterface()
     settings = BlurringAsAServiceSettings.get_settings()
 
-    data_to_blur = Input(
+    mounted_root_folder = Input(
         type=AssetTypes.URI_FOLDER,
-        path=settings["inference_pipeline"]["inputs"]["data_to_blur"],
+        path=settings["inference_pipeline"]["inputs"]["folder"],
```

Comment:
Did you rename data_to_blur to folder?
In config.example.yml it is still data_to_blur.
Also, can we rename this? maybe root_folder? since I saw you have mounted_root_folder later on